### PR TITLE
feat(PageSingleFeature)!: add option for accent eyebrow

### PR DESCRIPTION
**Pull request by aresnik11**
### Overview

Adds an `accentEyebrow` boolean prop which changes the eyebrow text color to `#1557FF` (as requested by design). When not provided or falsy, uses the default color for that color mode.

### PR Checklist

- [x] Related to designs: https://www.figma.com/file/J5yB1jyELiNt3Vb47T9GcE/Job-Readiness?node-id=2%3A43
- [x] Related to JIRA ticket: [REACH-1377](https://codecademy.atlassian.net/browse/REACH-1377)
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.104-alpha.39b94b.0<br/>@codecademy/gamut-labs@24.4.2-alpha.39b94b.0<br/>@codecademy/styleguide@47.4.2-alpha.39b94b.0<br/></pre>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.5.105-alpha.8ba0e5.0<br/>@codecademy/gamut-labs@24.4.3-alpha.8ba0e5.0<br/>@codecademy/styleguide@47.5.1-alpha.8ba0e5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://615ce88285989e246730783e--gamut-preview.netlify.app>https://615ce88285989e246730783e--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/615ce88285989e246730783e>Deploy Logs</a></p>

File: packages/gamut-labs/src/LandingPage/PageSingleFeature.tsx [hunk 1]
```diff
@@ -37,6 +37,10 @@ export type PageSingleFeatureProps = BaseProps & {
    * Eyebrow text shown above title
    */
   eyebrow?: string;
+  /**
+   * If the eyebrow text color should be an accent blue or the standard color for the current color mode
+   */
+  accentEyebrow?: boolean;
```
[the remark this answers is not in the document]
Comment (JoshuaKGoldberg) [reply packages/gamut-labs/src/LandingPage/PageSingleFeature.tsx:43]: Word, yes!

**Pull request merged**